packages/exoflex/src/components/Badge.tsx

```diff
@@ -55,7 +57,7 @@ export default function Badge(props: BadgeProps) {
           },
         ]}
         preset={textPreset}
-        testID={testID}
+        testID={testID as string}
```

Comment:
Looks a bit weird, what was the original type for `testID`?

Reply:
<img width="788" alt="Screen Shot 2023-02-01 at 12 24 32 PM" src="https://user-images.githubusercontent.com/25154563/215958712-107cc3c0-0cb6-482e-988d-c501eb451f0e.png">
<img width="1512" alt="Screen Shot 2023-02-01 at 12 25 04 PM" src="https://user-images.githubusercontent.com/25154563/215958790-36d53aa6-356a-4679-9244-861a64b6a62c.png">

Since we based the props from `PaperBadge`, and it's Props type based on Animated.Text, I think it counts `testID` as `Animated.Value` here and it raises `TypeError`.

Reply:
That is super weird, why would you want animated value in the test id.
Just to be on the safe side, could you try using String constructor instead of casting the type? Better safe on the runtime rather than sorry it crashed because of bad test id.
```suggestion
        testID={String(testID)}
```

Reply:
Got it, will do!